src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -107,6 +115,12 @@ class SharedLinkSection extends React.Component<Props, State> {
             this.setState({ isAutoCreatingSharedLink: true });
             addSharedLink();
         }
+
+        // if ESL ftux tooltip is showing on initial mount, we call onShow
+        if (sharedLinkEditTooltipTargetingApi && sharedLinkEditTooltipTargetingApi.canShow) {
```

Comment:
I think you need to check also `allowedPermissionLevels.includes(CAN_EDIT)` here too. Otherwise, it may lead to onShow being called, when it actually cannot show.

Reply:
Ah good catch! Will put in a fix PR for this, thanks for being thorough and finding this